frontend/src/components/dialogs/AddLinkDialog.tsx

```diff
@@ -80,12 +81,21 @@ export function AddLinkDialog({ target }: AddLinkDialogProps): React.ReactElemen
   const refreshCompounds = useCompoundsRefresher();
   const editing =
     (target.kind === "methodology" && target.object.item.confluence) ||
-    (target.kind === "compound" && target.object.item.chemSoft);
+    (target.kind === "compound-chemsoft" && target.object.item.chemSoft) ||
+    (target.kind === "compound-mde" && target.object.item.mde);
   const editingLink =
-    target.kind === "methodology" ? target.object.item.confluence : target.object.item.chemSoft;
```

Comment:
I didn't like this code before (when I wrote it), now I don't like it's even more :putya: